addons/wowl/doc/dropdown_component.md

```diff
@@ -0,0 +1,186 @@
+# Dropdown Component
+
+## Overview
+
+As dropdowns are common in Odoo, we decided to make a generic dropdown component.
+
+It contains all the logic of what you can usually expect a dropdown to behave.
```

Comment:
i'd rather detail which are those since I don't really know how I would expect a dropdown to behave.
I'd write
``` 
It currently supports
- Clicking on its target toggles the list
- .....
```